deploy/charts/emqx-operator/templates/admission-webhooks/patch-job/conversion/job.yaml

```diff
@@ -53,7 +59,8 @@ spec:
                           "namespace": "{{ .Release.Namespace }}",
                           "name": "{{ include "emqx-operator.fullname" . }}-webhook-service",
                           "path": "/convert"
-                        }
+                        },
+                        "caBundle": '$data'
```

Comment:
I think we should patch this field only if `admissionWebhooks.cert.certgen.enable = true`